storagenode/storageusage/storageusage_test.go

```diff
@@ -0,0 +1,161 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storageusage_test
+
+import (
+	"math"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testrand"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/storagenode"
+	"storj.io/storj/storagenode/storagenodedb/storagenodedbtest"
+	"storj.io/storj/storagenode/storageusage"
+)
+
+func TestStorageUsage(t *testing.T) {
+	const (
+		satelliteNum = 10
+		days         = 30
+	)
+
+	now := time.Now().UTC()
```

Comment:
For tests don't use `UTC()` the databases should handle the conversion internally. It's clearer to enforce this in the database implementation rather than outside.